packages/math/src/Transform.ts

```diff
@@ -9,6 +9,32 @@ import { Matrix } from './Matrix';
  */
 export class Transform
 {
+    /**
+     * A default (identity) transform
+     *
+     * @static
+     * @constant
+     * @member {PIXI.Transform}
+     */
+    public static IDENTITY = new Transform();
```

Comment:
This should be `public static readonly` and the doc comment doesn't need any of those values they are all in the Typescript.